components/docs/CodeBlock.jsx

```diff
@@ -0,0 +1,68 @@
+import React from 'react'
```

Comment:
I think all these files have  Apache Licence. 
* https://github.com/zentered/next-product-docs/blob/main/LICENSE

And I think we should add a `(c) Copright Zentered` at the top of each of the imported files with a link to the upstream version on GitHub.